modules/attributes/pages/document-attributes-reference.adoc

```diff
@@ -17,1100 +17,1057 @@ Need to update the compatibility guide with:
 
 Built-in document attributes can be set and unset using an attribute entry.
 
-Automatically Set:: The attribute is set automatically by Asciidoctor and assigned its default value at processing time if you don't set it explicitly in the API, command line, or document.
-Automatically set attributes can be unset with the `!`.
+Set By Default:: The attribute is set automatically by Asciidoctor and assigned a `*default*` value if not otherwise set.
+The default value is indicated in `bold` (e.g., `*skip*`).
+(A built-in attribute which is not set may still have an implied value, but that value won't be accessible using an attribute reference).
```

Comment:
This is pretty hard to understand and it might do more harm than good to mention that at this point.
Maybe we should keep it for later once we know what an implied value is?

Reply:
Now that we have another place for this information later on, I think we can drop the mention here.